src/wfqc/pckg_dev.py

```diff
@@ -0,0 +1,348 @@
+
+
+import pandas as pd
+import json
+import numpy as np
+import json 
+import random
+from sklearn.model_selection import train_test_split
+
+
+def stratified_split(data, test_size=0.2, random_state=42):
```

Comment:
Even though this module might not be part of the final package it is a good habit to document methods, it is easier to debug, review and improve later on, even if the documentation is one sentence.